client/signup/steps/domains/index.jsx

```diff
@@ -330,13 +330,14 @@ const submitDomainStepSelection = ( suggestion, section ) => {
 };
 
 export default connect(
-	state => ( {
+	( state, ownProps ) => ( {
```

Comment:
Since we're only using one prop, you could destructure here,

```js
( state, { initialContext } ) => ( {
```

which is a bit more declarative.